lte/gateway/c/session_manager/test/Matchers.h

```diff
@@ -301,24 +301,37 @@ MATCHER_P(CheckDeactivateFlows, imsi, "") {
 MATCHER_P(CheckSrvResponse, expected_response, "") {
   auto actual_response = static_cast<const SetSMSessionContextAccess>(arg);
 
-  auto expected_session_ambr = &(expected_response->rat_specific_context()
+  auto expected_subscribed_qos = &(expected_response->rat_specific_context()
                                      .m5g_session_context_rsp()
-                                     .session_ambr());
+                                     .subscribed_qos());
 
-  auto actual_response_ambr = &(actual_response.rat_specific_context()
+  auto actual_subscribed_qos = &(actual_response.rat_specific_context()
                                     .m5g_session_context_rsp()
-                                    .session_ambr());
+                                    .subscribed_qos());
 
   auto unit_res =
-      (expected_session_ambr->br_unit() == actual_response_ambr->br_unit());
+	(expected_subscribed_qos->br_unit() == actual_subscribed_qos->br_unit());
 
-  auto ul_ambr_res = (expected_session_ambr->max_bandwidth_ul() ==
-                      actual_response_ambr->max_bandwidth_ul());
+  auto ul_ambr_res = (expected_subscribed_qos->apn_ambr_ul() ==
+                      actual_subscribed_qos->apn_ambr_ul());
 
-  auto dl_ambr_res = (expected_session_ambr->max_bandwidth_dl() ==
-                      actual_response_ambr->max_bandwidth_dl());
+  auto dl_ambr_res = (expected_subscribed_qos->apn_ambr_dl() ==
+                      actual_subscribed_qos->apn_ambr_dl());
 
-  return (unit_res && ul_ambr_res && dl_ambr_res);
+  auto priority_lev_res = (expected_subscribed_qos->priority_level() ==
+                           actual_subscribed_qos->priority_level());
+
+  auto pre_cap_res = (expected_subscribed_qos->preemption_capability() ==
+                      actual_subscribed_qos->preemption_capability());
+
+  auto pre_vuln_res = (expected_subscribed_qos->preemption_vulnerability() ==
+                       actual_subscribed_qos->preemption_vulnerability());
+
+  auto qos_class_id_res = (expected_subscribed_qos->qos_class_id() ==
+                           actual_subscribed_qos->qos_class_id());
```

Comment:
As a general comment, we should specify types instead of auto. In this case, immediately seeing that the type of these is boolean makes it a lot easier to read. 

Also, I think these names could be a bit more descriptive. `unit_res` -> `unit_is_equal`